src/client/pages/OfferNew/Checkout/CheckoutContent.tsx

```diff
@@ -45,19 +44,12 @@ const StartDateLabel = styled.p`
   line-height: 1;
 `
 
-const InsuranceTypeLabel = styled.div`
-  font-size: 0.75rem;
-  color: ${colorsV3.gray500};
-  text-transform: uppercase;
-`
-
-const InsuranceType = styled.div`
+const InsuranceHeading = styled.h3`
   font-size: 1.5rem;
-  font-family: ${fonts.FAVORIT};
-  line-height: 1;
+  line-height: 2rem;
 
-  @media (min-width: 40rem) {
-    font-size: 2rem;
+  span {
+    display: block;
```

Comment:
Maybe just use a `div` since that has `display: block` as default?

Reply:
This is intentional, since a `div` is not allowed as a child of a heading element

Reply:
Ah, I see! Didn't think about that. 🤕  And what about just wrapping each of the text element in an `<InsuranceHeading>` tag? Since they also have `display: block` as default?

Reply:
No worries! That would work for sure👌  But for the sake of semantics, it could be bad to split a title into multiple headings. In this particular page and with a h3 I assume it wouldn't matter much :D But let's say it was an h1, then it would be worse since you should only have on h1 per page